test/Docker.DotNet.Tests/ISystemOperations.Tests.cs

```diff
@@ -15,6 +17,15 @@ public ISystemOperationsTests()
             _client = new DockerClientConfiguration(new Uri("npipe://./pipe/docker_engine")).CreateClient();
         }
 
+        [Fact]
+        public void DockerService_IsRunning()
+        {
+            var services = ServiceController.GetServices();

```

Comment:
Only one question. Why did you remove the:
```C#
using (var services = ...)
{
  //...
}
```
`Component` is disposable so shouldn't we do that?